Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Sliders/SliderSounds.cs

```diff
@@ -39,6 +39,22 @@ public class SliderSounds : MonoBehaviour
         [SerializeField]
         private float tickEvery = 0.1f;
 
+        [HideInInspector]
+        public float TickEvery
+        {
+            get
+            {
+                return tickEvery;
+            }
+            set
+            {
+                if (value <= 1.0f || value >= 0.0f)
```

Comment:
```suggestion
                if (value <= 1.0f && value >= 0.0f)
```

Reply:
I think that with the OR any number would work (i.e. -1 is < 1.0f, so the thing would evaluate out to true and then set it to that invalid value.